index.js

```diff
@@ -15,13 +16,14 @@ function simpleGet (opts, cb) {
   if (opts.headers == null) opts.headers = {}
   if (opts.maxRedirects == null) opts.maxRedirects = 10
 
-  var body = opts.json ? JSON.stringify(opts.body) : opts.body
+  var body = opts.json ? JSON.stringify(opts.body) : opts.form ? qs.stringify(opts.form) : opts.body
```

Comment:
I assume this option is modeled after the same option in `request`. If `opts.form` is already a query string then we should not call `qs.stringify`.